yt/yt/core/misc/async_slru_cache.h

```diff
@@ -344,6 +352,8 @@ class TAsyncSlruCacheBase
 
         void UpdateWeight(const TKey& key, i64 newWeight);
 
+        void UpdateCookieWeight(const TKey& key, i64 cookieWeight);
```

Comment:
UpdateWeight and UpdateCookieWeight don't look good together.
1) argument names are different, should we make them same?
2) should we rename UpdateWeight -> UpdateItemWeight?

Reply:
I'll rename argument.

These are methods of ghost-shard - not a public api